fbc-stage-index/release-resources/ec-policy.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: EnterpriseContractPolicy
+metadata:
+  name: e2e-fbc-stage-index-policy
+  namespace: managed-release-team-tenant
+spec:
+  description: >-
+    Includes rules for levels 1, 2 & 3 of SLSA v0.1.
+  publicKey: "k8s://openshift-pipelines/public-key"
+  sources:
+    - name: Default
+      policy:
+        - github.com/enterprise-contract/ec-policies//policy/lib
+        - github.com/enterprise-contract/ec-policies//policy/release
+      data:
+        - github.com/enterprise-contract/ec-policies//data
+  configuration:
+    include:
```

Comment:
@scoheb EC has specific policy rules related with FBC, refer: https://enterprisecontract.dev/docs/ec-policies/release_policy.html#labels_package. so I recommand to include policy label.fbc_required_labels,label.fbc_optional_labels and label.fbc_disallowed_inherited_labels here.

Reply:
and `fbc` rules are not included in the `slsa1, slsa2, slsa3` collections?

Reply:
no, these rules fall into `redhat` collection.

Reply:
thanks! I'll create a PR that uses the @redhat collection